sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-keyvault-secrets/README.md

```diff
@@ -70,6 +70,19 @@ azure.keyvault.secret-service-version=specify secretServiceVersion value(fill wi
 # azure.keyvault.secret-keys=yourSecretPropertyName1,yourSecretPropertyName2
 ```
 
+`azure.keyvault.authority-host`
```

Comment:
These two properties should also be added to this [doc](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-keyvault-secrets).